doc/code/sf_configuration.rst

```diff
@@ -36,6 +36,12 @@ and has the following format:
 Configuration options
 ---------------------
 
+``[api]``
+^^^^^^^^^
+
+Settings for the Xanadu cloud platform.
+
+
```

Comment:
This renders like this right now:
![image](https://user-images.githubusercontent.com/24476053/79242559-7be88a00-7e42-11ea-9ff1-9df91369e35c.png)

(Minor suggestion): How about indenting the options? (Such that "Settings for the..." is as is now, and "**authentication_token**" is the first to be indented one level)

Reply:
Good question, I'm not sure if that is possible? In ReST, indenting blocks of text is how you markup block quotations, so it would affect the formatting